### PR TITLE
Organize configuration.md

### PR DESCRIPTION
**Pull request by itaigilo**
## Change Description

Making `configuration.md` more readable.
No lines or info was added / updated / removed.

### CODE REVIEW

File: docs/reference/configuration.md
Comment (Jonathan-Rosenberg) [inline docs/reference/configuration.md:1]: Please add newlines after headings

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -157,74 +170,125 @@ This reference uses `.` to denote the nesting of values.
...
+### installation
 * `installation.user_name` `(string : )` - When specified, an initial admin user will be created when the server is first run. Works only when `database.type` is set to local. Requires `installation.access_key_id` and `installation.secret_access_key`. 
 * `installation.access_key_id` `(string : )` - Admin's initial access key id (used once in the initial setup process)
 * `installation.secret_access_key` `(string : )` - Admin's initial secret access key (used once in the initial setup process)
+
+### usage_report
 * `usage_report.enabled` `(bool : false)` - Store API and Gateway usage reports into key-value store.
 * `usage_report.flush_interval` `(duration : 5m)` - Sets interval for flushing in-memory usage data to key-value store.
 
+### more
```
Comment (Jonathan-Rosenberg) [inline docs/reference/configuration.md:285]: I think that `listen_address` should appear higher in the configurations...

File: docs/reference/configuration.md [hunk 2]
```diff
@@ -157,74 +170,125 @@ This reference uses `.` to denote the nesting of values.
...
+* `listen_address` `(string : "0.0.0.0:8000")` - A `<host>:<port>` structured string representing the address to listen on
+* `ui.enabled` `(bool: true)` - Whether to server the embedded UI from the binary
+* `security.audit_check_interval` `(duration : 24h)` - Duration in which we check for security audit.
+* `ugc.prepare_max_file_size` `(int: 125829120)` - Uncommitted garbage collection prepare request, limit the produced file maximum size
+* `ugc.prepare_interval` `(duraction: 1m)` - Uncommitted garbage collection prepare request, limit produce time to interval
```
Comment (Jonathan-Rosenberg) [inline docs/reference/configuration.md:290]: Those could be under `### garbage collection`

**Review by Jonathan-Rosenberg — CHANGES_REQUESTED** [review]
Thank you for that!
Some comments added.

File: docs/reference/configuration.md (hunk 1 above)
Comment (itaigilo) [reply docs/reference/configuration.md]: That's the only root-level value 🤷
Moved it up.

**Review by Jonathan-Rosenberg — APPROVED** [review]
Lovely!

**Pull request merged**